transactions/setNFTInfo_Dandy.cdc

```diff
@@ -0,0 +1,20 @@
+import Admin from "../contracts/Admin.cdc"
+import NFTRegistry from "../contracts/NFTRegistry.cdc"
+import Dandy from "../contracts/NFTRegistry.cdc"
+
+
+transaction() {
+
+    let adminRef : &Admin.AdminProxy
+
+    prepare(account: AuthAccount){
+        self.adminRef = account.borrow<&Admin.AdminProxy>(from: Admin.AdminProxyStoragePath) ?? panic("Cannot borrow Admin Reference.")
+        
+    }
+
+    execute{
+        let type: Type = Type<@Dandy.Collection>()
```

Comment:
```suggestion
        let type: Type = Type<@Dandy.NFT>()
```

Reply:
We want to signal that items of the type Dandy are stored in this registry slot so it is the NFT type that we need here.